<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-UMG1TUF" failures="1" tests="1" name="Default test(failed)(failed)(failed)(failed)" time="0.053" errors="0" timestamp="2022-05-06T11:14:33 IST">
  <testcase classname="testngdisscusion.FailedTestCaseExecution" name="login" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testngdisscusion.FailedTestCaseExecution.login(FailedTestCaseExecution.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="testngdisscusion.FailedTestCaseExecution" name="contacts" time="0.0">
    <ignored/>
  </testcase> <!-- contacts -->
  <testcase classname="testngdisscusion.FailedTestCaseExecution" name="dashboard" time="0.0">
    <ignored/>
  </testcase> <!-- dashboard -->
  <testcase classname="testngdisscusion.FailedTestCaseExecution" name="profile" time="0.0">
    <ignored/>
  </testcase> <!-- profile -->
</testsuite> <!-- Default test(failed)(failed)(failed)(failed) -->
